src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -99,9 +99,9 @@ public void initialize() {
             } else {
                 this.participantList = storageParticipantList.get();
                 int largestIdUsed = participantList.list().stream()
-                        .map(participant -> ((Entity) participant).getId().getNumber())
```

Comment:
Id is exposed on entity so we dont need to typecast

Reply:
yup so i removed the casting